draft-ietf-rats-corim.md

```diff
@@ -821,7 +832,7 @@ An instance carries a unique identifier that is reliably bound to a Target Envir
 that is an instance of the Attester.
 
 The types defined for an instance identifier are CBOR tagged expressions of
-UEID, UUID, or cryptographic key identifier.
+UEID, UUID, variable-length opaque byte string, or cryptographic key identifier.
```

Comment:
Similar entry is needed in the class identifier as well!

Reply:
see [6fee7e3](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/188/commits/6fee7e39d2eaa8cf792443ede914bfcf53d8b71a)